sos/__init__.py

```diff
@@ -23,7 +23,7 @@
 
 import gettext
 
-__version__ = "3.2"
+__version__ = "3.3"
```

Comment:
Hi,

Le 12/07/2016 13:32, Bryn M. Reeves a écrit :
> In sos/__init__.py <https://github.com/sosreport/sos/pull/851#discussion_r70421193>:
> 
>> @@ -23,7 +23,7 @@
>>  
>>  import gettext
>>  
>> -__version__ = "3.2"
>> +__version__ = "3.3"
> 
> Committing this in the tree doesn't do any harm (although it's arguable that
> having |__init__.py| in git is a bug itself - it is a build artifact generated
> from |__init__.py.in| at build time - it's just been in there for so long I
> never saw a good reason to remove it).
> 
> If you're getting builds that don't update it however that is a problem -
> possibly something missing in the |setup.py| build (the |Makefile| build does do
> this substitution correctly).
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sosreport/sos/pull/851/files/8006a73a0d8ec078cad3b878c329a2c2e5680276#r70421193>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABezp2GzDhhkPFgRdqJMa2izoeISf2mUks5qU3tbgaJpZM4JKR10>.
> 

Well since this is the first time we change major version on the debian build
there might be problems with the process. Le me look at this.

Kind regards,

...Louis
-- 
Louis Bouchard
Software engineer, Cloud & Sustaining eng.
Canonical Ltd
Ubuntu developer                       Debian Maintainer
GPG : 429D 7A3B DD05 B6F8 AF63  B9C4 8B3D 867C 823E 7A61